src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -11,15 +11,15 @@
         </section>
         <ul>
           <li>
-            <v-icon v-if="hasApplicant" color="green darken-2" class="dir-valid">mdi-check</v-icon>
-            <v-icon v-else color="red">mdi-close</v-icon>
-            <!-- <v-icon>mdi-circle-small</v-icon> -->
+            <v-icon v-if="isApplicantPerson" color="green darken-2" class="dir-valid">mdi-check</v-icon>
+            <v-icon v-else-if="!isApplicantPerson"  color="red">mdi-close</v-icon>
+            <v-icon v-else>mdi-circle-small</v-icon>
```

Comment:
I'm confused by this. It looks like
```
if True then ...
if False then ...
else ...
```

You have 3 scenarios but the logic only supports 2 of them.

The screenshots mostly look good, except the one with both a red X and a green check -- should not have the red X if a person or org is selected.

Reply:
Or did you mean `applicantPersons`?

Reply:
I clearly did not think this through earlier. Updated it